src/components/Genre/Genre.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./Genre.css"
+
+const Genre = (props) => {
+    return (
+        <p className='Genre'>{props.genre}</p>
+    )
+}
```

Comment:
This component is itty-bitty enough that I wonder if its really worth having it be its own component.  Just something to consider (I see Chris mused something similar).  That said, if you haven't refactored to get rid of this component, I'm not sure its something you _need_ to do at this point.   Maybe a piece to reflect on after to see if you would still break this out separately next time - if it felt worth it or not.  <Great interview fodder!